src/components/common/Onboarding/wizardSteps/CreateColony/validation.ts

```diff
@@ -116,6 +117,11 @@ export const createTokenValidationSchema = object({
       then: (schema) =>
         schema
           .max(MAX_TOKEN_NAME, '')
+          .test(
+            'isNotOnlySpaces',
+            formatMessage({ id: 'error.tokenNameInvalid' }),
+            (value) => value.trim().length > 0,
+          )
           .required(formatMessage({ id: 'error.tokenNameRequired' })),
```

Comment:
@bassgeta however in order to not trigger the validation upon blur when there is no value, I believe the 
```
         .test(
            'isNotOnlySpaces',
            formatMessage({ id: 'error.tokenNameInvalid' }),
            (value) => value.trim().length > 0,
          )
```
has to be after `.required(formatMessage({ id: 'error.tokenNameRequired' }))`